cg/meta/orders/status.py

```diff
@@ -391,6 +394,14 @@ def store_fastq_samples(
         self.status.add_commit(new_samples)
         return new_samples
 
+    @staticmethod
+    def if_type_is_wgs_then_we_should_do_mip_dna_analysis_for_maf(application_version):
```

Comment:
longest name in cg 😄

Reply:
Could you type hint input and output?

Reply:
Would not: `if_type_wgs_do_mip_dna_analysis_for_maf` suffice? 😄

Reply:
good suggestion, fixed